chord_metadata_service/restapi/api_renderers.py

```diff
@@ -117,18 +118,19 @@ def generate_csv_response(data, filename, columns):
     return response
 
 
-def render_individual_age(individual):
-    if "time_at_last_encounter" not in individual:
+def render_age(individual: Dict[str, Any], time_key: str) -> Optional[str]:
```

Comment:
this shouldn't be called `individual`, since it can also be a biosample. Give it a more generic name like `item`